res/css/photon/radio-button.css

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.photon-radio {
+  border: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background: none;
+  background-color: var(--grey-90-a10);
+  border-radius: 8px;
+}
+
+/* Use this class with the photon-radio for the canonical photon styling. */
+.photon-radio-default {
+  width: 16px;
+  height: 16px;
+  margin: 0 4px;
+}
+
+/* Use this class with the photon-radio for the custom micro variant styling. */
+.photon-radio-micro {
+  width: 13px;
+  height: 13px;
+  margin: 0 3px;
```

Comment:
I see this rule is overriden by a `margin: 0 4px` in `.stackSettingsFilterInput`. I don't mind which value you end of choosing, but in the end we should remove the one from `.stackSettingsFilterInput` for ease and consistency.